app/src/main/java/com/example/notes2text/usecases/FileEditingUseCases/OpenTextEditorInteractor.java

```diff
@@ -0,0 +1,25 @@
+package com.example.notes2text.usecases.FileEditingUseCases;
+
+import java.io.File;
+
+public class OpenTextEditorInteractor implements OpenTextEditorBoundary {
+    File lastFile;
+
+    String lastFileRoot;
+
+
+    public OpenTextEditorInteractor(File file) {
+        this.lastFile = file;
+        this.lastFileRoot = file.getAbsolutePath();
+    }
+
+    // Extract text from the txt file
+    public String extractContent() {
```

Comment:
would this retain any bolding and italicizing that you made if you open a previously edited file?